app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] When a user signs up they should be required to confirm their email before signing in. Kindly add the `:confirmable` module and ensure users confirm their accounts after signing up.

Reply:
@julie-ify done with this change